packages/fswatch.rb

```diff
@@ -3,30 +3,25 @@
 class Fswatch < Package
   description 'fswatch is a file change monitor that receives notifications when the contents of the specified files or directories are modified.'
   homepage 'https://github.com/emcrisostomo/fswatch'
-  version '1.11.2'
+  version '1.17.1'
   license 'GPL-3+'
   compatibility 'all'
```

Comment:
Shouldn't this have changed to `compatibility 'x86_64 aarch64 armv7l'`?

Reply:
i686 (if any?) users should be able to build it from the source as the package still supports i686

Reply:
Then why didn't we generate a binary? ;)